### ISSUE TITLE
Dedicated primitive (e.g. `Fused`) for fused convolutions?

### ISSUE THREAD

**Issue by atamazov** [ROCm/MIOpen#1213]
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/4e61a3ebdfe2b07b4d331cc46832eb4a6b49941c/src/solver.cpp#L264

Can we introduce some dedicated primitive (e.g. `Fused`) for fused convolutions and use it in the registry? This would help to avoid hacks like this: https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1212/files/db05b058f77cc838f5bbafc93fe08fd8e20c713d..f26346d12ba5f2ee472d26d7e6c207325bbe6233#r723662796

**Comment by ppanchad-amd** [comment]
@DrizztDoUrden Has this been resolved? Thanks!

**Comment by DrizztDoUrden** [comment]
@ppanchad-amd  No, but I can take it if we consider it valuable and urgent. Do we? It is a trivial thing to do, but merging any PR takes quite some time.

**Comment by JonathanLichtnerAMD** [comment]
Closing since MIOpen is being migrated to [ROCm/rocm-libraries](https://github.com/ROCm/rocm-libraries).  Please re-open there if you believe this is still an issue.
